packages/@aws-cdk/aws-glue/test/table.test.ts

```diff
@@ -6,6 +6,7 @@ import * as cdk from '@aws-cdk/core';
 import * as cxapi from '@aws-cdk/cx-api';
 import { testFutureBehavior } from 'cdk-build-tools/lib/feature-flag';
 import * as glue from '../lib';
+import { CfnTable } from '../lib/glue.generated';
```

Comment:
We should be able to write a super simple eslint rule here to enforce this - https://github.com/aws/aws-cdk/blob/master/tools/eslint-plugin-cdk/lib/rules/construct-import-order.ts

Happy to bootstrap one if you like.